homeworks/sasha.demidenko_SashaDemi/GamesProject/games.html

```diff
@@ -62,247 +62,41 @@ <h1 class="games_h1">
-                            </p>
-                            <p style="margin: 0">
-                                <span class="game_subtitle--style">Platform:</span>PC (Windows)
-                            </p>
-                            <p style="margin: 0">
-                                <span class="game_subtitle--style">Publisher:</span>Activision Blizzard King
-                            </p>
-                            <p style="margin: 0">
-                                <span class="game_subtitle--style">Developer:</span>Blizzard Entertainment
-                            </p>
-                            <p style="margin: 0">
-                                <span class="game_subtitle--style">Release_date:</span>2022-10-04
-                            </p>
-                        </div>
-                    </div>
-                </article>
+                <template data-type="card-template">
+                    <article data-is-new="true" data-is-old="false">
+                        <div class="game_container">
+                            <div class="game_title-name">
+                                <div class="title-img">
+                                    <img src="Images/Overwatch_img.jpg" alt="no img" data-type="thumbnail">
+                                </div>
+                                <h2 class="title-name" data-type="title">Overwatch 2</h2>
+                                <div class="title-description" data-type="short_description">
+                                    Big changes come to the Overwatch formula in this sequel...
+                                </div>
+                            </div>
+                            <div class="game_list">
+                                <p style="margin: 0">
+                                    <span class="game_subtitle--style" data-type="genre"></span>
+                                </p>
+                                <p style="margin: 0">
+                                    <span class="game_subtitle--style" data-type="platform"></span>
+                                </p>
+                                <p style="margin: 0">
+                                    <span class="game_subtitle--style" data-type="publisher"></span>
+                                </p>
+                                <p style="margin: 0">
+                                    <span class="game_subtitle--style" data-type="developer"></span>
+                                </p>
+                                <p style="margin: 0">
+                                    <span class="game_subtitle--style" data-type="release_date"></span>
+                                </p>
+                            </div>
+                        </div>
+                    </article>
+                </template>
             </div>
+            <ul data-type="cards-container" class="cards_wrapper"></ul>
         </main>
+        <script src="scripts/games.js"></script>
```

Comment:
Hey, @SashaDemi, looks like you didn't add the `games.js` file to the commit, cause I don't see it 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/4c4b847f-fd81-4f69-bfc4-e57ab60c5d9a)